tests/aead_tests.rs

```diff
@@ -148,7 +150,7 @@ fn test_aead(aead_alg: &'static aead::Algorithm, file_path: &str) {
             257, // 16 AES blocks or 8 ChaCha20 blocks, plus 1.
         ];
 
-        let mut more_comprehensive_in_prefix_lengths = [0; 4096];
+        let mut more_comprehensive_in_prefix_lengths = [constants::UNINITIALIZED_USIZE; 4096];
```

Comment:
I don't think we need to worry about this, because this isn't being passed to a function under test. So, I suggest just reverting it back to 0.